api/models/calendarEntry.js

```diff
@@ -34,33 +34,41 @@ const calendarEntrySchema = new mongoose.Schema(
       type: Boolean,
       required: true,
     },
-    recurringEventId: {
-      type: mongoose.Schema.Types.ObjectId,
-      ref: "CalendarEntry",
+    recurrenceEndsUtc: {
+      type: Date,
```

Comment:
I didn't realize this was `Date` type. That makes me think there is more simplifcation that can be done with how we handle dates, to pass in Dates rather than stringified dates using `toISOString()` or other functions.

Reply:
When it comes in, it comes in as an ISO string which is correct. Using the 'Date' type here makes it so that Mongoose casts the object to a Date internally. More on that here: https://mongoosejs.com/docs/tutorials/dates.html

I think because the date info comes over the wire it has to be in a string format to work correctly, then we can change it to a Date on the backend side.